---
external_plugins:
  allow_plugins: true
  trace_plugin_allocations: true
  paths: [hydra_ros]
